sdk/test-utils/perf/GettingStarted.md

```diff
@@ -88,52 +86,11 @@ To add perf tests for the `sdk/<service>/<service-sdk>` package, follow the step
          }
     ```
 
-### [For perf-testing track 1 version of the same package](#for-perf-testing-track-1-version-of-the-same-package)
-
-(_Skip this section if your service does not have or does not care about a track-1 version._)
-
-1. If there is an old major version of your package that needs to be compared, create the folder as `sdk/<service>/perf-tests/<service-sdk>-track-1`
-
-2. It is expected that the track-1 perf tests are counterparts of track-2 tests, so they need to have the same names as specified in the track-2 tests for convenience.
-
-3. Add a `package.json` such as [example-track-1-perf-package.json](https://github.com/Azure/azure-sdk-for-js/blob/fe9b1e5a50946f53b6491d7f67b2420d8ee1b229/sdk/storage/perf-tests/storage-blob-track-1/package.json) at `sdk/<service>/perf-tests/<service-sdk>` folder.
-
-   Make sure to import your `<service-sdk>` and the `test-utils-perf` project.
-
-   ```json
-     "dependencies": {
-        "@azure/<service-sdk>": "^<latest-track-1-version>",
-        "@azure-tools/test-perf": "file:../../../test-utils/perf/azure-test-utils-perf-1.0.0.tgz",
-      }
-   ```
-
-   Set the name of the package and mark it as private.
-
-   ```json
-    "name": "@azure-tests/perf-<service-sdk>-track-1",
-    "sdk-type": "perf-test"
-    "private": true,
-   ```
-
-   _Note: Track-1 packages will not be managed by `rush`, instead `npm` will be used to manage/run the track-1 tests, you can copy the readme such as the [storage-blob-perf-tests-track-1-readme](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/storage/perf-tests/storage-blob-track-1/README.md) for instructions._
-
-   Make sure to add the "setup" step in package.json.
-
-   ```json
-       "setup": "node ../../../../common/tools/perf-tests-track-1-setup.js",
-   ```
-
-4. Run `rush update` followed by `npm run setup` to be able to use the perf framework for track-1 perf tests.
-
-   _`npm run setup` installs the dependencies specified in `package.json`_
-
-5. Repeat the step 6 from the previous section for the track-1 too to get the `tsconfig.json`, `sample.env`(and `.env`) files.
-
 ## [Writing perf tests](#writing-perf-tests)
 
 ### [Entry Point](#entry-point)
 
-Add an `index.spec.ts` at `sdk/<service>/perf-tests/<service-sdk>/test/`.
+Add an `index.ts` at `sdk/<service>/<service-sdk>-perf-tests/src/`.
```

Comment:
Also should make a note in the perf thing that it is executing node dist/esm/index.js for example.

Reply:
There's a section about running with `npm run perf-test:node`